mksnapshot.sh

```diff
@@ -9,41 +9,43 @@
 
 echo "RUN: $0"
 if [ $# -lt 1 ]; then
-	echo "Usage: $0 [branch]"
-	echo "Where [branch] is the path under /bzr/ to the branch to snapshot."
+	echo "Usage: $0 [branch] [tag]"
+	echo "Where [branch] is the name of the branch to snapshot."
+	echo "Where [tag] is the name of the tag on that branch to snapshot."
```

Comment:
You seem to be thinking in bzr terms. In git, tags are global -- they do not belong to a branch. In other words, specifying a tag alone is sufficient to identify a code snapshot. Do we need the branch name here? See `git describe` below for a related discussion.